cmd/mattermost/commands/user.go

```diff
@@ -345,18 +345,28 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	}
 
 	if systemAdmin {
-		if _, err := a.UpdateUserRoles(ruser.Id, "system_user system_admin", false); err != nil {
```

Comment:
I think this change is not related, and not necessary. This already was idiomatic error handling, if you only going to handle the error, this is the way to go, all in the same line.

Reply:
done